### PR TITLE
RHDEVDOCS 5971 Release Notes for Pipelines 1.14.2

### PR DESCRIPTION
**Pull request by mramendi**
Version(s):

please cp to pipelines-docs-1.14

Issue:

RHDEVDOCS 5971

Link to docs preview:

https://73527--ocpdocs-pr.netlify.app/openshift-pipelines/latest/about/op-release-notes#release-notes-1-14-2_op-release-notes

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/op-release-notes-1-14.adoc [hunk 1]
```diff
@@ -302,3 +302,15 @@ With this update, {pipelines-title} General Availability (GA) 1.14.1 is availabl
...
+
+[id="release-notes-1-14-2_{context}"]
+== Release notes for {pipelines-title} General Availability 1.14.2
+
+With this update, {pipelines-title} General Availability (GA) 1.14.2 is available on {OCP} 4.12 and later versions.
+
+[id="fixed-issues-1-14-2_{context}"]
+=== Fixed issues
+
+* Before this update, when you started a pipeline run using {pac}, {tekton-results} did not store information about this pipeline run. Because of this issue, the web console plugin did not include the pipeline run in the execution statistics display. With this plugin, {tekton-results} stores information about {pac} pipeline runs and these pipeline runs are included in the execution statistics display.
```
Comment (ppitonak) [inline modules/op-release-notes-1-14.adoc:314]: `With this pluging` ... did you mean to write `plugin` here?

**Review by ppitonak — APPROVED** [review]

File: modules/op-release-notes-1-14.adoc [hunk 2]
```diff
@@ -302,3 +302,15 @@ With this update, {pipelines-title} General Availability (GA) 1.14.1 is availabl
...
+== Release notes for {pipelines-title} General Availability 1.14.2
+
+With this update, {pipelines-title} General Availability (GA) 1.14.2 is available on {OCP} 4.12 and later versions.
+
+[id="fixed-issues-1-14-2_{context}"]
+=== Fixed issues
+
+* Before this update, when you started a pipeline run using {pac}, {tekton-results} did not store information about this pipeline run. Because of this issue, the web console plugin did not include the pipeline run in the execution statistics display. With this update, {tekton-results} stores information about {pac} pipeline runs and these pipeline runs are included in the execution statistics display.
+
+* Before this update, when you started many pipeline runs using {pac} at the same time and these pipelines runs includes a `max-keep-run` annotation, the {pac} watcher component failed to process some of the pending pipeline runs and they remained in a pending state. With this update, {pac} pipeline runs are processed correctly.
```
Comment (kcarmichael08) [inline modules/op-release-notes-1-14.adoc:316]:
```suggestion
* Before this update, when you started many pipeline runs using {pac} at the same time and these pipelines runs included a `max-keep-run` annotation, the {pac} watcher component failed to process some of the pending pipeline runs and they remained in a pending state. With this update, {pac} pipeline runs are processed correctly.
```

**Review by kcarmichael08 — COMMENTED** [review]
One issue, otherwise lgtm!

**Pull request merged**